cli/wallet/wallet.go

```diff
@@ -96,9 +100,13 @@ func walletAction(c *cli.Context) error {
 				fmt.Fprintln(os.Stderr, err)
 				os.Exit(1)
 			}
+			var vbs bool
```

Comment:
What is the vbs used to and how to protect the private key be show from remotely? I mean remote connect the nknd with nknc --ip --port

Reply:
nknc wallet always use local wallet.dat, --ip will be ignored